src/components/grid/hovered_cell_store.ts

```diff
@@ -1,25 +1,59 @@
+import { setColorAlpha } from "../../helpers";
+import { PositionMap } from "../../plugins/ui_core_views/cell_evaluation/position_map";
 import { SpreadsheetStore } from "../../stores";
-import { Command, Position } from "../../types";
+import { Color, Command, Position } from "../../types";
 
 export class HoveredCellStore extends SpreadsheetStore {
-  mutators = ["clear", "hover"] as const;
+  mutators = ["clear", "hover", "debouncedHover"] as const;
```

Comment:
I'm not confortable with the distinction between `debouncedHover` and `hover `, what do you think of introducing a second store ?
- The first could be named something like `ImmediateHoveredCellStore`, and should be responsible to keep track of hovered cell when the mouse enter the cell.
- The second (could be named `DelayedHoveredCellStore`) would be responsible to track the hovered cell when the mouse stays in the cell ?

The two would expose `hover` mutator and `col` and `row` as accessor. It should be the role of the caller to know which store we would like to use.

What do you think ?

Reply:
Also, it's strange to have the computation of the overlays based on a table in such a generic store. Also, `col` and `row` seems not used outside the store. We could make a store `HoveredTableStore` instead of `ImmediateHoveredCellStore` which only expose `hoverCell` mutator and `overlayColors` accessor.